src/lib/api/EventAPI.ts

```diff
@@ -145,3 +146,28 @@ export const uploadEventImage = async (
     },
   });
 };
+
+export const awardBonusPoints = async (
+  token: string,
+  user: string,
+  points: number,
+  description: string
+): Promise<CreateBonusResponse> => {
+  const requestUrl = `${config.api.baseUrl}${config.api.endpoints.admin.bonus}`;
+
+  const requestBody: CreateBonusRequest = {
+    bonus: {
+      users: [user],
```

Comment:
Maybe you should have `awardBonusPoints` accept a list of users since the backend supports it, and the function returns a list of emails rather than just one